packages/components/Organisms/SiteFooter/FooterNavigation.vue

```diff
@@ -4,7 +4,7 @@
       <li class="rpl-footer-nav__menu-item" :class="{'rpl-footer-nav__menu-item--parent': item.children}" :key="index" >
         <h2 class="rpl-footer-nav__heading" @click="toggle(item, index, $event)">
           <rpl-link :href="item.url" :target="item.target">{{ item.text }}</rpl-link>
-          <rpl-icon :symbol="visibleItemIndex == index ? 'up' : 'down'" color="secondary" size="m" v-if="(minimize && item.children)" />
+          <rpl-icon :symbol="visibleItemIndex == index ? 'up' : 'down'" color="white" size="m" v-if="(minimize && item.children)" />
```

Comment:
Hi @alan-cole @dylankelly this is what I mentioned last time in the icon meeting, this hardcoded `color` can't be customised by Sass variable, need to use custom css to override it which is increased maintenance work.